pages/home/client.js

```diff
@@ -1,10 +1,20 @@
 var photoGrid = require('../../lib/photos')
+var sentTpl = require('./sent.jade')
 
 module.exports = function (page) {
-  page('/', enter)
+  page('/', enter, sent)
 }
 
 function enter (ctx, next) {
+  if (ctx.querystring === 'sent=true') return next()
```

Comment:
@bmordan this'll prevent the photoGrid function from being called if the user sends a message. Is that deliberate?